sdk/compute/Azure.ResourceManager.Compute/tests/Generated/Samples/Sample_CapacityReservationCollection.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Compute.Models;
+
+namespace Azure.ResourceManager.Compute
+{
+    public partial class Sample_CapacityReservationCollection
+    {
+        // Create or update a capacity reservation .
+        [NUnit.Framework.Test]
+        [NUnit.Framework.Ignore("Only verifying that the sample builds")]
+        public async Task CreateOrUpdate_CreateOrUpdateACapacityReservation()
+        {
+            // Generated from example definition: specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/examples/capacityReservationExamples/CapacityReservation_CreateOrUpdate.json
+            // this example is just showing the usage of "CapacityReservations_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+
+            // authenticate your client
+            ArmClient client = new ArmClient(new DefaultAzureCredential());
+
+            // this example assumes you already have this CapacityReservationGroupResource created on azure
+            // for more information of creating CapacityReservationGroupResource, please refer to the document of CapacityReservationGroupResource
+            string subscriptionId = "{subscription-id}";
+            string resourceGroupName = "myResourceGroup";
+            string capacityReservationGroupName = "myCapacityReservationGroup";
+            ResourceIdentifier capacityReservationGroupResourceId = Compute.CapacityReservationGroupResource.CreateResourceIdentifier(subscriptionId, resourceGroupName, capacityReservationGroupName);
+            Compute.CapacityReservationGroupResource capacityReservationGroup = client.GetCapacityReservationGroupResource(capacityReservationGroupResourceId);
+
+            // get the collection of this CapacityReservationResource
+            Compute.CapacityReservationCollection collection = capacityReservationGroup.GetCapacityReservations();
```

Comment:
We cannot get rid of the `Compute.` namesapce prefix?

Reply:
The change is ongoing in the auto-PR. Will update this soon

Reply:
Just merge from main and regenerate. Now the `Compute.` prefix is removed